ansible/roles/docker/templates/ubi81.Dockerfile.j2

```diff
@@ -27,6 +27,13 @@ RUN dnf install --disableplugin=subscription-manager -y \
       procps-ng \
     && dnf --disableplugin=subscription-manager clean all
 
+RUN dnf install --disableplugin=subscription-manager -y \
+      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }/os/Packages/gcc-toolset-10-gcc-10.3.1-1.2.el8_5.{{ ansible_architecture }.rpm \
+      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }/os/Packages/gcc-toolset-10-gcc-c++-10.3.1-1.2.el8_5.{{ ansible_architecture }.rpm \
+      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }/os/Packages/gcc-toolset-10-libstdc++-devel-10.3.1-1.2.el8_5.i686.rpm \
+      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }/os/Packages/gcc-toolset-10-libstdc++-devel-10.3.1-1.2.el8_5.{{ ansible_architecture }.rpm \
+      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }/os/Packages/gcc-toolset-10-runtime-10.1-0.el8.{{ ansible_architecture }.rpm
```

Comment:
Missing closing `}`.
```suggestion
      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }}/os/Packages/gcc-toolset-10-gcc-10.3.1-1.2.el8_5.{{ ansible_architecture }}.rpm \
      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }}/os/Packages/gcc-toolset-10-gcc-c++-10.3.1-1.2.el8_5.{{ ansible_architecture }}.rpm \
      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }}/os/Packages/gcc-toolset-10-libstdc++-devel-10.3.1-1.2.el8_5.{{ ansible_architecture }}.rpm \
      http://mirror.centos.org/centos/8-stream/AppStream/{{ ansible_architecture }}/os/Packages/gcc-toolset-10-runtime-10.1-0.el8.{{ ansible_architecture }.rpm
```

We don't need the i686 package as we won't be cross-compiling in this container.